.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDatePeriodParser.cs

```diff
@@ -415,6 +437,20 @@ private DateTimeResolutionResult ParseComplexDatePeriod(string text, DateObject
 
                     if (endResolution.Success)
                     {
+                        // When start or end is ambiguous it is better to resolve it to the type of the unambiguous extraction
+                        if (isAmbiguousStart && isSpecificDate)
```

Comment:
Can you add an example here in the comment?

Reply:
added.

Reply:
The idea was to add the comment explaining what happens if only "mar" is mentioned. But I won't block the merge on this.